language/doublecmd.be.po

```diff
@@ -3,17 +3,19 @@ msgstr ""
 "Project-Id-Version: b5aaebc75354984d7cee90405a1f6642\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2008-01-17 23:08+0300\n"
-"PO-Revision-Date: 2020-05-30 08:13\n"
+"PO-Revision-Date: 2021-08-02 10:35\n"
 "Last-Translator: Full Name <email@address>\n"
 "Language-Team: Belarusian\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Native-Language: Беларуская\n"
+"X-Native-Language: English\n"
```

Comment:
Why you changed it? It should be language name at native language.

Reply:
Sorry, auto